docs/containers.conf.5.md

```diff
@@ -289,6 +289,12 @@ is imposed.
 
 Copy the content from the underlying image into the newly created volume when the container is created instead of when it is started. If `false`, the container engine will not copy the content until the container is started. Setting it to `true` may have negative performance implications.
 
+**privileged**=false
+
+Give extended privileges to this container. A privileged container turns off the security features that isolate the container from the host. Dropped Capabilities, limited devices, read-only mount points, Apparmor/SELinux separation, and Seccomp filters are all disabled.
+
+Rootless containers cannot have more privileges than the user that launched them.
+
```

Comment:
nit: we can also add "containers running using in a user namespace", they are limited in the same way as rootless containers.

Reply:
TY!